test/regtest.py

```diff
@@ -14,7 +19,7 @@
 
 def red(text):
```

Comment:
Should we keep the `red` and `green` functions? They're no longer being used. I tried to utilize them, but the logging class doesn't have an elegant way of writing colored output to log files AND console.